src/app/schedule-section.tsx

```diff
@@ -0,0 +1,93 @@
+import Image from "next/image"
+
+const schedule=[
+    {
+        title:"Opening Ceremony",
+        time:"10:00 AM",
+        duration:"20 min",
+        description:"Check in starting at 9:00 AM"
+    },
+    {
+        title:"Networking",
+        time:"10:20 AM",
+        duration:"75 min",
+        description:"Learn and practice the whole shabang of networking!"
+    },
+    {
+        title:"Lunch",
+        time:"11:35 AM",
+        duration:"30 min",
+        description:"Eat food - pizza."
+    },
+    {
+        title:"Presenting",
+        time:"12:10 PM",
+        duration:"75 min",
+        description:"Learn to deliver gnarly presentations"
+    },
+    {
+        title:"Pitch Competition",
+        time:"1:35 PM",
+        duration:"150 min",
+        description:"Pitch to judges, get the prizes"
+    },
+    {
+        title:"Dinner",
+        time:"4:10 PM",
+        duration:"30 min",
+        description:"Get filled with Chick Fil A"
+    },
+    {
+        title:"Financial Literacy",
+        time:"4:45 PM",
+        duration:"30 min",
+        description:"Learn to reduce losses"
+    },
+    {
+        title:"Closing Ceremony",
+        time:"5:20 PM",
+        duration:"30 min",
+        description:"Wind down, winners announced, and reflecting on the fun-filled day"
+    },
+    
+];
+
+export function ScheduleSection(){
+
+    return (
+        <div id="schedule" className="delimitationred xl:px-[180px] lg:px-[50px] md:px-[20px] relative flex flex-col items-center lg:justify-between lg:items-center">
+            {schedule.map((element, index) => <Leaf key={element.title} keyName={element.title} title={element.title} time={element.time} duration={element.duration} description={element.description} index={index}/>)}
+        </div>
+    );
+}
+
+export function Leaf(props: {keyName: string, title: string, time: string,  duration: string, description: string, index: number}){
+    return (
+        <>
+            <li  key={props.keyName+"topspace"} className={"flex  w-full text-white flex-row-reverse"}>
+                <div className={"w-[50%] flex border-white-16 flex-row border-l-8 px-[2%]"}>
```

Comment:
Don't use `{"string"}` for raw strings; instead just put them in quotes `className='the-value-goes-here'`

Reply:
yes